sdk/keyvault/keyvault-keys/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Release History
 
+## 4.0.0-preview.9 (2019-10-31)
+- deleteKey is now out of the public API. Use beginDeleteKey instead. beginDeleteKey returns a Long Running Operation Poller (from our package @azure/core-lro).
```

Comment:
should we refer to it as `Long Running Operation Poller` or just `Poller`? The former seems a bit wordy

Reply:
I think a compromise could be "returns a Poller (from our package @azure/core-lro) that managed the long running operation."